src/components/ArticleBanner/styles.css

```diff
@@ -21,24 +21,26 @@
   color:#a5b1b7;
 }
 .articles__banner__links {
-  transform: translateX(50%);
```

Comment:
why do you use `translateX` when you can use either `flex` or tailwind's `grid`?

Reply:
I will try `gird` as flex did  not wok cause the `flex-direction` is `column` and the last child will not be aligned at the bottom of the parent div

Reply:
Thanks @Nilegfx . It was a good idea and clear HTML, less CSS. 😄 
**_My flat_** is that I tried to adapt to the current HTML structure.